addons/mail/static/src/models/user.js

```diff
@@ -43,9 +43,9 @@ registerModel({
                 model: 'res.users',
                 method: 'read',
                 args: [ids],
+                fields: fields,
```

Comment:
Hello, this is not correct, this param is not supported here. I believe field should be moved into `args`, or better adapt `return ormService.read(model, args[0], args.length > 1 ? args[1] : undefined, kwargs);` to also look into `kwargs.fields` if `args[1]` is not defined.

Reply:
issue probably since #99018

Reply:
> Hello, this is not correct, this param is not supported here. I believe `fields` should be moved into `args`, or better adapt `return ormService.read(model, args[0], args.length > 1 ? args[1] : undefined, kwargs);` to also look into `kwargs.fields` if `args[1]` is not defined.

done. thank you

Reply:
But it would be better to fix the real issue here instead of adapting calls
`return ormService.read(model, args[0], args.length > 1 ? args[1] : kwargs['fields'], kwargs);`

Reply:
> But it would be better to fix the real issue here instead of adapting calls `return ormService.read(model, args[0], args.length > 1 ? args[1] : kwargs.fields, kwargs);`

Done.  I remove the fields from the kwargs otherwise a traceback occurs. OK for you now?